lib/kvdb/ikvdb.c

```diff
@@ -1279,53 +1289,31 @@ kvdb_kvs_destroy(struct kvdb_kvs *kvs)
 static merr_t
 ikvdb_cndb_open(struct ikvdb_impl *self, u64 *seqno, u64 *ingestid, u64 *txhorizon)
 {
-    merr_t           err = 0;
-    int              i;
-    struct kvdb_kvs *kvs;
+    merr_t            err = 0;
+    struct kvdb_kvs **kvsp;
 
     err = cndb_open(
         self->ikdb_mp,
-        self->ikdb_read_only,
-        &self->ikdb_seqno,
-        self->ikdb_rp.cndb_entries,
         self->ikdb_cndb_oid1,
         self->ikdb_cndb_oid2,
-        &self->ikdb_health,
+        self->ikdb_read_only,
         &self->ikdb_rp,
+        &self->ikdb_health,
         &self->ikdb_cndb);
     if (ev(err))
-        goto err_exit;
+        return err;
 
     err = cndb_replay(self->ikdb_cndb, seqno, ingestid, txhorizon);
```

Comment:
Does it make sense to make replay internal to CNDB and do it as part of cndb_open?

Reply:
Yes, that would be cleaner. Let me take a look.

Reply:
Future work! :)